src/elements/content-sidebar/activity-feed/task-new/TaskDueDate.js

```diff
@@ -1,10 +1,10 @@
 // @flow
 import * as React from 'react';
-import classNames from 'classnames';
 import { FormattedMessage } from 'react-intl';
 
 import messages from './messages';
 import { ReadableTime } from '../../../../components/time';
+import LabelPill from '../../../../components/label-pill';
```

Comment:
Minor, but we usually try to alphabetize imports in groups (third-party first, relative imports, types, scss).